tests/bluetooth/tester/src/btp_vcp.c

```diff
@@ -0,0 +1,544 @@
+/* vcp.c - Bluetooth VCP Tester */
+
+/*
+ * Copyright (c) 2022 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/bluetooth/bluetooth.h>
+
+#include <zephyr/bluetooth/testing.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/bluetooth/audio/vcp.h>
+#include <zephyr/bluetooth/audio/aics.h>
+#include "zephyr/bluetooth/audio/vocs.h"
+#include <app_keys.h>
+
+#include <zephyr/logging/log.h>
+#define LOG_MODULE_NAME bttester_vcp
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#include "btp/btp.h"
+
+#define CONTROLLER_INDEX 0
+
+struct bt_vcp_vol_rend_register_param vcp_register_param;
+struct bt_vcp_included included;
+static bool g_aics_active = 1;
```

Comment:
What is the purpose of this?

Reply:
`g_aics_active` will be removed, but I think we need `included` they are set in `tester_init_vcp`, and then they are needed e.g - bt_vocs_description_set(). I also wanted to make register parameters global thinking about future use, this way if there will be other parameters on init someone can change them easily.

Reply:
Sure; my comment should be been more explicit: I was actually only asking about `g_aics_active` :D